docs: replace certificates in pkcs12 format (#5528)
strimzi/strimzi-kafka-operator: documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

[id='installing-your-own-ca-certificates-{context}']
= Installing your own CA certificates

[role="_abstract"]
This procedure describes how to install your own CA certificates and keys instead of using the CA certificates and private keys generated by the Cluster Operator.

The Cluster Operator automatically generates and renews the following secrets:

`_CLUSTER-NAME_-cluster-ca`:: The cluster secret that contains the private key for the cluster CA.

> `_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers.

Reviewer: ```suggestion
`_CLUSTER-NAME_-cluster-ca-cert`:: The cluster secret that contains a cluster CA certificate. The certificate contains a public key to validate the identity of Kafka brokers.
```

`_CLUSTER-NAME_-clients-ca`:: The client secret that contains the private key for the client CA.

> `_CLUSTER-NAME_-clients-ca-cert`:: The client secret contains a client CA certificate. The certificate contains a public key to validate the identity of clients accessing the Kafka brokers.

Reviewer: ```suggestion
`_CLUSTER-NAME_-clients-ca-cert`:: The client secret that contains a client CA certificate. The certificate contains a public key to validate the identity of clients accessing the Kafka brokers.
```

2. One or more intermediate CAs
3. The root CA
+
** All CAs in the chain should be configured using the X509v3 Basic Constraints extension. Basic Constraints limit the path length of a certificate chain.
* The OpenSSL TLS management tool for converting certificates.

.Before you begin
The Cluster Operator generates the following files for the `_CLUSTER-NAME_-cluster-ca-cert` secret:

* `ca.crt` cluster certificate in PEM format
* `ca.p12` cluster certificate in PKCS #12 format
* `ca.password` to access the PKCS #12 file

> If you are using a service registry, make sure you include all three files in your secret.
> The service registry can then use a certificate in the appropriate format to connect securely with Kafka.

Reviewer 2: I don't think this should be here. Not all service registries require PKCS12 files and not all applications which need PKCS12 files are service registries. Maybe we can use something like this?

```suggestion
Some applications cannot use the PEM certificates directly and support only the PKCS #12 format.
To make it easier to use them, you can also add the cluster certificate in PKCS #12 format.
```

Author: Better. I thought I needed to say something specifically about service registries. 
Changed to:

> Some applications cannot use PEM certificates and support only PKCS #12 certificates. You can also add your own cluster certificate in PKCS #12 format.
